include/puppies/Dwarf.hpp

```diff
@@ -295,7 +295,7 @@ void ToolsMappingBody::windowEvent([[maybe_unused]] window_t *sender, GUI_event_
     uint16_t get_heatbreak_fan_pwr();
 
     inline uint8_t get_dwarf_nr() const {
-        return dwarf_nr;
+        return dwarf_nr - 1;
```

Comment:
Disrepance between the public getter and private member is even worse than the current state. If we wanna do this, we will need to go further

Reply:
Alternatively, we could rename `get_dwarf_nr` to `dwarf_index`

Reply:
Done.